.github/workflows/pull-request-ci.yaml

```diff
@@ -284,6 +284,13 @@ jobs:
 
       # Run smoke tests
       - run: make smoke-tests
+        id: smoke-tests
+        timeout-minutes: 10
+        continue-on-error: true
+
+      - name: Re-rune smoke tests
```

Comment:
I'm not completely convinced of this, but feel free to mark as resolved. If `make kubecf-wait` is not enough, we should either fix that or kubecf. Just out of curiosity, how many times have you seen the need for re-running smokes on the runs?

Reply:
I'm not convinced either, but then I noticed similar failures in concourse runs.  Of the last 5 runs (10 clusters, because diego + eirini), it might have fixed one of those?

I'm okay with dropping it.